net-wireless/uhd/uhd-4.1.0.5.ebuild

```diff
@@ -139,9 +138,3 @@ src_install() {
 	insinto /usr/share/${PN}
 	doins -r "${WORKDIR}/images"
 }
-
-src_test() {
```

Comment:
wait, is this just gone now? Do tests work now without?

Reply:
Yup. The benefit of upstream fixing that line I had to sed.  It was apparently fixed a while ago and I didn't notice when my issue was closed.  Details in the commit removing the sed iirc